scylla_pg_core/src/bin/db_migrate.rs

```diff
@@ -36,10 +37,16 @@ async fn run_migrations() {
 
 async fn try_create_db() -> Result<(), tokio_postgres::Error> {
     dotenv::dotenv().ok();
+    let is_env = std::env::var("DB_ENV").unwrap_or_default();
```

Comment:
Two comments on this:-
- We should avoid getting `env` variables in inner functions, it makes it hard to test them isolated, it's hard to  know which env variables are used where. Instead why not map it in the main function to corresponding `config` and pass it into the function as args?
- Why not do this check in the main function before calling `try_create_db()`? Right now we are not able to do it, because we are loading from the `.env` file in the inner function `try_create_db`
- And also because we are loading them in the inner functions, we are forced to load the `env` using `dotenv` twice. Once in `try_create_db` and again in `run_migrations`